hdl/syn/afc_v3/dbe_bpm2_bo_sirius/build_synthesis_sdb.sh

```diff
@@ -8,7 +8,7 @@ set -u
 # Maximum of 16 chars
 SYNTH_INFO_PROJECT="bpm-gw-bo-sirius"
 SYNTH_INFO_TOOL="VIVADO"
-SYNTH_INFO_VER=$(vivado -version | head -n 1 | cut -d' ' -f2 | cut -d 'v' -f2)
+SYNTH_INFO_VER=$(vivado -version | grep 'Vivado v[0-9]\{4\}.*' | head -n 1 | cut -d' ' -f2 | cut -d 'v' -f2)
```

Comment:
We shouldn't need `head -n 1` anymore, right?

Reply:
Well, if for some motive Vivado prints duplicated lines, then `head -n 1` would make sure that only the first match is selected. But reading the `man grep` manual, it seems that `grep` can be instructed to get the first match and then exit with `-m 1`. I'll use this solution.